python_examples/intervention_example/disease_testing_parameters.json

```diff
@@ -501,8 +501,8 @@
         "policy_duration": 365,
         "case_threshold": 0,
         "testing_capacity": [100,1000],
-        "false_positive": [0.1, 0.1],
-        "false_negative": [0.1, 0.1],
+        "false_positive": [0.023, 0.0032],
+        "false_negative": [0.2, 0.222],
```

Comment:
These look good, can I just check you've references where you get these numbers from in the wiki?